source/containers/Form/Form.js

```diff
@@ -16,16 +16,19 @@ const FormContainer = styled.View`
  * is a tool to help you solve the problem of allowing end-users to interact with the
  * data and modify the data in your application.
  */
-function Form({ startAt, steps, firstName, onClose }) {
+function Form({ startAt, steps, firstName, onClose, userAnswer }) {
   const initialState = {
     counter: startAt,
     steps,
     user: {
       firstName,
     },
+    answerPerCase: userAnswer, // set the initial answer fetched per usercase
```

Comment:
The form do not know anything about a case, so maybe rename this to simply answers or values?

Reply:
or formAnswers?